dev_tools/input/Fct.json

```diff
@@ -0,0 +1,16277 @@
+[
```

Comment:
This is just the JSON equivalent of the `Fct.csv` file that we've been provided, right?